packages/hagrid/hagrid/deps.py

```diff
@@ -479,17 +484,55 @@ def check_docker_version() -> Optional[str]:
     return version
 
 
-def docker_running() -> bool:
+def docker_running() -> Tuple[bool, str]:
     try:
-        cmd = "docker info"
+        cmd = "docker --version"
```

Comment:
@IonesioJunior we want to use `docker info` as it checks if docker is running or not on windows and mac and starts docker incase of ubuntu. `docker --version` doesn't detect that. 

I think we need to fix the order of checking sudo and docker running status. First check docker has sudo status and then check docker running status.